tcp.go

```diff
@@ -114,6 +113,7 @@ func probeTCP(ctx context.Context, target string, module Module, registry *prome
 				}
 			}
 			if scanner.Err() != nil {
+				log.Errorf(scanner.Err().Error())
```

Comment:
The first parameter to Errorf is the format string, so put something in here to give the user context